agreement/cryptoVerifier_test.go

```diff
@@ -385,3 +387,24 @@ func BenchmarkCryptoVerifierBundleVertification(b *testing.B) {
 		<-c
 	}
 }
+
+// TestCryptoVerifierVerificationFailures tests to see that the cryptoVerifier.VerifyVote returns an error in the vote response
+// when being unable to enqueue a vote.
+func TestCryptoVerifierVerificationFailures(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
+	mainPool := execpool.MakePool(t)
+	defer mainPool.Shutdown()
+
+	voteVerifier := MakeAsyncVoteVerifier(&expiredExecPool{mainPool})
+	defer voteVerifier.Quit()
+
+	cryptoVerifier := makeCryptoVerifier(nil, nil, voteVerifier, logging.TestingLog(t))
+	defer cryptoVerifier.Quit()
+
+	cryptoVerifier.VerifyVote(context.Background(), cryptoVoteRequest{message: message{Tag: protocol.AgreementVoteTag}, Round: basics.Round(8)})
+	// read the failed response from VerifiedVotes:
+	votesout := cryptoVerifier.VerifiedVotes()
+	voteResponse := <-votesout
+	require.Equal(t, context.Canceled, voteResponse.err)
```

Comment:
Arguably you would want voteResponse.cancelled = true to match a context.Canceled err.

Reply:
(or maybe preserving current behavior of no response at all is fine, as I mentioned earlier)